docker/docker-compose.yml

```diff
@@ -0,0 +1,55 @@
+database:
+  image: "quay.io/galexrt/zulip-postgresql-tsearchextras:latest"
+  environment:
+    DB_NAME: zulip
+    DB_USER: zulip
+    DB_PASS: zulip
+  volumes:
+    - "/opt/docker/zulip/postgresql/data:/var/lib/postgresql:rw"
+memcached:
+  image: "quay.io/sameersbn/memcached:latest"
+  restart: always
+rabbitmq:
+  image: "docker.io/rabbitmq:3.5.5"
```

Comment:
@djdefi What docker version are you running? I have no problems with docker version 1.8.3 and 1.7.1, running it using `docker-compose up` and `docker run docker.io/rabbitmq:3.5.5` works for me.